db/seeds.rb

```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
```

Comment:
Delete commented-out code (I know this a placeholder, but could cause confusion now that you have real code below)